lib/ansible/plugins/cliconf/ios.py

```diff
@@ -70,13 +70,13 @@ def get_config(self, source='running', flags=None):
     @enable_mode
     def edit_config(self, command):
         for cmd in chain(['configure terminal'], to_list(command), ['end']):
-            try:
-                cmd = json.loads(cmd)
+            if cmd[0] == '{':
```

Comment:
can we use `ast.literal_eval()` here to convert string to valid dict (that has double-quoted strings )

Reply:
I thought about it, but doing evals make me nervous.
However, just googled and it seems literal eval != eval, meaning it won't allow a malicious code to be evaluated:

https://stackoverflow.com/questions/15197673/using-pythons-eval-vs-ast-literal-eval

So yeah, it's more pythonic, will amend, thx.